src/constants/actions.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // action types
+  TRANSITION: 'ACTION_TYPE/TRANSITION',
+  METADATA: 'ACTION_TYPE/METADATA',
+  PROGRESS: 'ACTION_TYPE/PROGRESS',
+  // transition actions
+  REJECT: 'TRANSITION/REJECT',
```

Comment:
`POST /pins/{pinId}/state_transition -d { "state": "rejected" }`

Other samples

`POST /pins/{pinId}/state_transition -d '{ "state": "assigned", "assignedDepartment": {departmentId} }'`
`POST /pins/{pinId/state_transition -d '{ "state": "processing", "processedBy": {userId} }'`

`POST /pins/{pinId}/metadata_change -d '{ tags: ["footpath", "dirty"] }'`

`POST /pins/{pinId}/progress_change -d '{ "text": "Fixed already!" }'`
`GET /pins/{pinId}/progress_change` (get an array of progress to be shown in timeline)
`POST /pins/{pinId}/progress_change/{progressId} -d '{ "text": "[Edit] Not fixed yet" }'`